src/components/DepositWidget/index.tsx

```diff
@@ -12,9 +12,8 @@ import { ZERO, MEDIA } from 'const'
 import { TokenBalanceDetails } from 'types'
 
 // Components
-import { CardTable } from 'components/Layout/Card'
+import { CardTable, CardWidgetWrapper } from 'components/Layout/Card'
```

Comment:
Oh, so that was the card layout you were talking about!

I was using the `CardTable` component and thought that was it.

Reply:
haha my bad ;)